packages/tasit-identity-contract/src/GnosisSafe.test.js

```diff
@@ -119,94 +128,106 @@ describe("GnosisSafe", () => {
       );
       await execTxAction.waitForNonceToUpdate();
 
-      const balanceAfterWithdraw = await provider.getBalance(
-        GNOSIS_SAFE_ADDRESS
-      );
-      expect(`${balanceAfterWithdraw}`).to.equal(`${ZERO}`);
+      const balance = await provider.getBalance(GNOSIS_SAFE_ADDRESS);
+      expect(`${balance}`).to.equal(`${ZERO}`);
     });
+  });
 
-    it("wallet owner should withdraw some ethers", async () => {
+  describe("test cases that needs ERC20 deposit to the wallet", async () => {
+    beforeEach("faucet", async () => {
+      const { address: anaAddress } = anaWallet;
+
+      erc20.setWallet(anaWallet);
```

Comment:
Should we call `ana` "minter" here instead? I think that makes it clearer why her wallet has mint permissions

Reply:
Is this thread still open or no?

Reply:
Extract to: https://github.com/tasitlabs/TasitSDK/issues/237

Reply:
I think that could be close since the thread is tracked by that new issue

Reply:
Okay thanks. Resolving.